android/src/main/java/com/apptentive/android/sdk/reactlibrary/ApptentiveModule.kt

```diff
@@ -151,24 +151,36 @@ class ApptentiveModule(private val reactContext: ReactApplicationContext) :
   // Delegated from addCustomPersonData(key, value)
   @ReactMethod
   fun addCustomPersonDataBoolean(key: String, value: Boolean, promise: Promise) {
-    Apptentive.addCustomPersonData(key, value)
-    promise.resolve(true)
+      try {
```

Comment:
This way is fine with me. In other plugins if I remember correctly, we utilized `isRegistered` flag

Reply:
Yeah, here we already had a lot of try catches so it just follows the code style